src/util/keys.ts

```diff
@@ -52,3 +52,10 @@ export const SkyBrowserInverseZoomDirectionKey =
   'Modules.SkyBrowser.InverseZoomDirection';
 export const SkyBrowserSpaceCraftAnimationTimeKey =
   'Modules.SkyBrowser.SpaceCraftAnimationTime';
+
+//Keys used by NightSkyPanel
+export const StarsDimInAtmosphereKey = 'Scene.Stars.Renderable.DimInAtmosphere';
+export const StarsRenderableKey = 'Scene.Stars.Renderable';
+export const StarLabelsRenderableKey = 'Scene.StarsLabels.Renderable';
+export const StarLabelsAlternateRenderableKey = 'Scene.StarLabelsAlternate.Renderable';
+export const MoonPerformShadingKey = 'Scene.Moon.Renderable.PerformShading';
```

Comment:
We've gone away from using keys like this, and instead just use the full URI directly in the code. This is more readable, and most property keys are only used in one place anyways. 

Please remove these variables and use the full string where you are using them instead :)

Reply:
Or create variables in the individual components where you use them multiple times